tests/playwright/tests/AddBitGet.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { test, expect } from '@playwright/test';
+
+//Add BITGET wallet 24 mnemonic + ensure address is correct
+
+test('BITGET', async ({ page }) => {
+  await page.goto('/');
+  await page.getByRole('button', { name: 'Get started' }).click();
+  await page.getByRole('button', { name: 'Existing Wallet Import wallet' }).click();
+  await page.getByLabel('1:', { exact: true }).click();
+  await page.getByLabel('1:', { exact: true }).fill(process.env.BITGET_MNEMONIC_24);
+  await page.getByRole('button', { name: 'Continue' }).click();
+  await expect(page.getByText('W5', { exact: true })).toBeVisible();
+  await expect(page.getByText('UQC2…1-Ly · 0 TON')).toBeVisible();
```

Comment:
This wallet have a ton amount
Should we just check wallet address and don't check an amount on ton?

Reply:
Exactly. Because this wallet is in-use one, as soon as TON amount changes - test will fail. Anyway the main point of checking is to ensure that addresses on all stages of the flow are the same.